2015-16/C01 Make Episode File.sps

```diff
@@ -373,7 +366,7 @@ sort cases by chi keydate1_dateformat.
 
 *  We are not currently including Social Care data for some years but we still want the variables for consistency.
 * Social Care.
- * String
+String
```

Comment:
This and the CH variables below should not need to be added in the 1516 file

Reply:
CH variables added back in 1516 file, not too sure about numeric variables. Can you please double check what i have added back in/deleted is correct